webcam_publisher_node:
  ros__parameters:
    topci_name: /webcam/image_raw
    camera_index: 0
    publish_rate: 30.0
